### DISCUSSION TITLE
good practices before starting to contribute to this organisation?

### DISCUSSION THREAD

**Discussion by rajgorsmit** [EddieHubCommunity/support#8666] [Q&A] [answered]
before starting contributing to this github organization i would like to know any good practices
- languages : python, c-sharp, javascript, c++
- frameworks : nextjs, nodejs - express, dotnet
- what i can do : full stack web dev, mobile application dev, neural networks & ml

**Comment by dinxsh** [comment, marked as answer]
hey @rajgorsmit, based on your skillset you should be able to contribute to most of the repos on [EddieHub](https://github.com/orgs/EddieHubCommunity/repositories) make sure to keep this points in mind before planning to contribute to any repository!!

1. **Understanding the Project:**
- Before contributing, thoroughly understand the project's goals, coding standards, and community guidelines. Review existing documentation and issues to identify areas where your contributions can add value.

2. **Communication:**

- Engage with the community through discussions, issue comments, and pull requests. Clarify doubts and seek feedback to ensure alignment with project objectives.

3. **Start Small:**

- Begin with small contributions such as documentation updates, bug fixes, or minor feature enhancements. This allows you to familiarize yourself with the project's codebase and processes gradually.

4. **Follow Contribution Guidelines:**

- Adhere to the organization's contribution guidelines, including coding standards, commit message format, and licensing requirements. This promotes consistency and facilitates smooth integration of your contributions.

5. **Create Quality Pull Requests:** 

- Submit well-documented pull requests that clearly explain the problem being addressed, the proposed solution, and any relevant context or dependencies. Include tests whenever applicable to maintain code quality and reliability.

lastly do join the [discord](https://discord.com/invite/jZQs6Wu) community to chat with the community!! 💬
